Purchases/Logging/Strings/AttributionStrings.swift

```diff
@@ -17,46 +17,118 @@ import Foundation
+    case attributes_sync_success(appUserID: String)
 
-    static let marking_attributes_synced = "Marking the following attributes as synced for App User ID: %@: %@"
+    case empty_subscriber_attributes
 
-    static let method_called = "%@ called"
-    static let networkuserid_required_for_appsflyer = "The parameter networkUserId is REQUIRED for AppsFlyer."
+    case marking_attributes_synced(appUserID: String, attributes: SubscriberAttributeDict)
 
-    static let no_instance_configured_caching_attribution = "There is no purchase instance configured, " +
-        "caching attribution"
+    case method_called(methodName: String)
 
-    static let instance_configured_posting_attribution = "There is a purchase instance configured, posting attribution"
+    case networkuserid_required_for_appsflyer
 
-    static let search_ads_attribution_cancelled_missing_att_framework = "Tried to post Apple Search Ads Attribution, " +
-        "but ATT Framework is required on this OS and it isn't included"
+    case no_instance_configured_caching_attribution
 
-    static let att_framework_present_but_couldnt_call_tracking_authorization_status = "ATT Framework was found but " +
-        "it didn't respond to authorization status selector!"
+    case instance_configured_posting_attribution
 
-    static let iad_framework_present_but_couldnt_call_request_attribution_details = "iAd Framework was found but " +
-        "it didn't respond to attribution details request!"
+    case search_ads_attribution_cancelled_missing_att_framework
 
-    static let search_ads_attribution_cancelled_missing_iad_framework = "Tried to post Apple Search Ads Attribution, " +
-        "but iAd Framework is is required for it and it isn't included"
+    case att_framework_present_but_couldnt_call_tracking_authorization_status
 
-    static let search_ads_attribution_cancelled_not_authorized = "Tried to post Apple Search Ads Attribution, but " +
-        "authorization hasn't been granted. Will automatically retry if authorization gets granted."
+    case iad_framework_present_but_couldnt_call_request_attribution_details
 
-    static let skip_same_attributes = "Attribution data is the same as latest. Skipping."
-    static let subscriber_attributes_error = "Subscriber attributes errors: %@"
-    static let unsynced_attributes_count = "Found %lu unsynced attributes for App User ID: %@"
-    static let unsynced_attributes = "Unsynced attributes: %@"
-    static let attribute_set_locally = "Attribute set locally: %@. It will be synced to the backend" +
-        "when the app backgrounds/foregrounds or when a purchase is made."
-    static let missing_advertiser_identifiers = "Attribution error: identifierForAdvertisers is missing"
-    static let missing_app_user_id = "Attribution error: can't post attribution, missing appUserId"
+    case search_ads_attribution_cancelled_missing_iad_framework
+
```

Comment:
style: I wouldn't keep extra lines between cases, but that's just my opinion

Reply:
This is from #816, I'll update it here, though. 
I am leaving the extra lines for the `var description`, since in that one it does get a little hard to read through without them.

Reply:
👍